services/terraform/terraform-base.js

```diff
@@ -0,0 +1,50 @@
+import Joi from 'joi'
+import { nonNegativeInteger } from '../validators.js'
+import { BaseJsonService } from '../index.js'
+
+const description =
+  '[Terraform Registry](https://registry.terraform.io) is an interactive resource for discovering a wide selection of integrations (providers), configuration packages (modules), and security rules (policies) for use with Terraform.'
+
+const schema = Joi.object({
+  data: Joi.object({
+    attributes: Joi.object({
+      month: nonNegativeInteger,
+      total: nonNegativeInteger,
+      week: nonNegativeInteger,
+      year: nonNegativeInteger,
+    }).required(),
+    id: Joi.string().required(),
+    type: Joi.string().required(),
```

Comment:
Looks like we don't actually use the `id` or `type` values.

If so, we can remove these two from the schema. We don't need to validate values we don't use: We should only fail if values we actually rely on to render the badge are missing from the response or don't match the expected format.